### PR TITLE
ocamlPackages.hxd: 0.3.1 -> 0.3.2

### PR DESCRIPTION
**Pull request by sternenseemann**
https://github.com/dinosaure/hxd/releases/tag/v0.3.2

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/ocaml-modules/hxd/default.nix [hunk 1]
```diff
@@ -5,22 +5,17 @@
 
 buildDunePackage rec {
   pname = "hxd";
-  version = "0.3.1";
+  version = "0.3.2";
 
   useDune2 = true;
 
   minimumOCamlVersion = "4.06";
```
Comment (superherointj) [inline pkgs/development/ocaml-modules/hxd/default.nix:12]:
```suggestion
  minimumOCamlVersion = "4.08";
```
https://github.com/ocaml/opam-repository/blob/master/packages/hxd/hxd.0.3.2/opam#L19

File: pkgs/development/ocaml-modules/hxd/default.nix [hunk 2]
```diff
@@ -5,22 +5,17 @@
```
Comment (vbgl) [inline pkgs/development/ocaml-modules/hxd/default.nix:5]: I think that `withLwt` and `ocaml` are no longer necessary here.

File: pkgs/development/ocaml-modules/hxd/default.nix (hunk 1 above)
Comment (vbgl) [reply pkgs/development/ocaml-modules/hxd/default.nix:12]: And spell it minim**al**.

File: pkgs/development/ocaml-modules/hxd/default.nix [hunk 3]
```diff
@@ -5,22 +5,17 @@
 
 buildDunePackage rec {
   pname = "hxd";
-  version = "0.3.1";
+  version = "0.3.2";
 
   useDune2 = true;
```
[the remark this answers is not in the document]
Comment (sternenseemann) [reply pkgs/development/ocaml-modules/hxd/default.nix]: done.

File: pkgs/development/ocaml-modules/hxd/default.nix (hunk 2 above)
Comment (sternenseemann) [reply pkgs/development/ocaml-modules/hxd/default.nix:5]: right, it's still optional, but we can just default to `true`.

**Pull request merged**